sklearn/feature_selection/base.py

```diff
@@ -120,3 +120,67 @@ def inverse_transform(self, X):
         Xt = np.zeros((X.shape[0], support.size), dtype=X.dtype)
         Xt[:, support] = X
         return Xt
+
+
+def feature_wise_scorer(score_func):
+    """ A wrapper function around score functions.
+
+    Parameters
+    ----------
+    score_func : callable
+        Function taking array(s) X and/or y, and returning a pair of arrays
+        (scores, pvalues) or a single array with scores.
```

Comment:
You're not documenting that the user has the option to pass in `**kwargs`, which alter the behavior of the `score_func`.